src/main/java/seedu/workbook/model/internship/util/StageUtil.java

```diff
@@ -0,0 +1,98 @@
+            "Once finished, remember to ALWAYS check that your code compiles and works for edge cases as well!"
+    );
+
+    private static final List<String> phoneInterviewTips = Arrays.asList(
+            "A recruiter is looking for the following indicators during the Phone Interview: Passion for tech/coding, Enthusiasm, Communication skills, Culture fit, Alignment with company mission/values",
+            "As such, focus on displaying your soft skills (Show not tell!)",
+            "Radiate excitement about the company or project, and that positivity will make a good first impression on your interviewer.",
+            "Be honest: What interests you the most about this job? What kinds of projects were you hoping to work on?",
+            "Do your research and tailor your answers to the exact position and company you’re interviewing for.",
+            "Look for opportunities to show your value alignment. Show that you support the mission of the organization."
+    );
+
+    private static final List<String> rejectedTips = Arrays.asList(
+            "Acknowledge your emotions and feelings about the rejection. Don’t worry, it is perfectly normal to feel this way but don’t give up!",
+            "Remind yourself that you are competent. Getting rejected does not mean you are not qualified, it could just mean that there was an applicant more suitable for the particular job! Capitalise on the skills and expertise that are unique to yourself!",
+            "Express gratitude by thanking the company for the opportunity and taking the time to know you better! If you are interested in applying to the company in the future, do also maintain a good impression and let them know about it! :)",
+            "Ask for feedback from the company so that you can do better the next time! It’ll be good to find out what went well, what could be improved on and how to go about improving them!",
+            "Most importantly, keep trying and move on to the next opportunity, they are plenty awaiting you! Sharpen, refine your skills before applying for internships again, you’ll be one step closer to landing the internship of your dream!"
+    );
+
+    private static final List<List<String>> listOfTips= Arrays.asList(
+            applicationSentTips,
+            onlineAssessmentTips,
+            technicalInterviewTips,
+            behavioralInterviewTips,
+            phoneInterviewTips,
+            rejectedTips
+    );
+
+    public static final Map<Stage, List<String>> tipsForStage = new HashMap<>();
+
+    /**
+     * Constructor for StageUtil class
+     * Populates the map whereby each stage is mapped to a list of tips.
+     */
+    public StageUtil() {
+        for (int i = 0; i < tipsForStage.size(); i++) {
+            tipsForStage.put(stageWithTips.get(i), listOfTips.get(i));
+        }
+    }
+
+    /**
+     * Returns a list of tips depending on the stage.
```

Comment:
Would be good to describe behaviour so that our frontend engineer knows to check for null if the stage does not have tips.
Something like "Returns a List of tips for the input stage, or `null` if none are found."